components/controls/edit/KmInputRichLstring.vue

```diff
@@ -104,6 +104,9 @@ export default {
         const clean = useKmStorage().cleanDocument({...this.binding});
         this.$emit('update:modelValue', clean);
     },
+    onFileUpload({file}){
```

Comment:
same, declare your events

Reply:
yes , i the previous PR declared all the events, but probably u looked at the latter PR first #24